libs/transactions-journey/internal/feature-transaction-view/src/lib/components/transactions-view/transactions-view.component.html

```diff
@@ -51,20 +72,46 @@
         </bb-badge-ui>
       </div>
 
-      <ul *ngIf="transactions$ | async as items; else loading" class="bb-list">
-        <li
-          *ngFor="let item of items | filterTransactions : filter"
-          class="bb-list__item"
-          data-role="transactions-view__item-container"
+      <ng-container *ngIf="transactions$ | async as items; else loading">
```

Comment:
`*ngIf` is deprecated.  We should be using `@if`.

Reply:
added